protocol/common/chat_entity.go

```diff
@@ -0,0 +1,20 @@
+package common
+
+import (
+	"crypto/ecdsa"
+
+	"github.com/golang/protobuf/proto"
+
+	"github.com/status-im/status-go/protocol/protobuf"
+)
+
+type ChatEntity interface {
```

Comment:
Could we add a comment here explaining how this is supposed to encompass `Message` and `EmojiReaction`?